utils/templates/termite.config.template

```diff
@@ -0,0 +1,32 @@
+[colors]
+# !!COL!{name}! colorscheme for Termite
+
+# 20% background transparency (requires a compositor)
+#background = rgba(63, 63, 63, 0.8)
+
+foreground = !!COL!{fg.srgb}!
+foreground_bold = !!COL!{br_white.srgb}!
+#foreground_dim = #888888
```

Comment:
can you explain what does Termite use this color for? I may be able to suggest what to assign here.

Reply:
You mean `foreground_dim`?

I've never actually seen someone use it in their config. I would say it would be `\033[2mSome Text"`: ie "dim or feint"

https://github.com/thestinger/vte-ng/blob/0.50.2-ng/src/vterowdata.h#L62
https://github.com/thestinger/vte-ng/blob/0.50.2-ng/src/vte.cc#L5953

Looking at this, RGB is not accepted.

https://github.com/thestinger/vte-ng/blob/0.50.2-ng/src/vte.cc#L8734

Reply:
ok. Interesting.